libcalico-go/lib/clientv3/interface.go

```diff
@@ -56,6 +56,8 @@ type Interface interface {
 	KubeControllersConfiguration() KubeControllersConfigurationInterface
 	// CalicoNodeStatus returns an interface for managing CalicoNodeStatus resources.
 	CalicoNodeStatus() CalicoNodeStatusInterface
+	// IPAMConfig returns an interface for managing IPAMConfig resources.
+	IPAMConfig() IPAMConfigInterface
```

Comment:
nit: should this be `IPAMConfiguration` to match the other naming conventions? It doesn't particularly matter that it's abbreviated other than it doesn't quite match.

Reply:
We try to keep libcalico-go files using the old name so that the name update has limited impact. Please see comment https://github.com/projectcalico/calico/pull/6393#discussion_r927814045